core/kazoo_number_manager/src/providers/knm_failover.erl

```diff
@@ -42,19 +39,19 @@ save(Number, _State) ->
 %% @end
 %%--------------------------------------------------------------------
 -spec delete(knm_number:knm_number()) -> knm_number:knm_number().
-delete(Number) ->
-    case feature(Number) =:= 'undefined' of
-        'true' -> Number;
-        'false' -> knm_services:deactivate_feature(Number, ?KEY)
+delete(N) ->
```

Comment:
This hunk seems unnecessary. Number is clearer than N, there's no change to functionality, so why the change?